src/encoder.rs

```diff
@@ -21,6 +24,80 @@ fn encode_vlq_diff(out: &mut String, a: u32, b: u32) {
     encode_vlq(out, i64::from(a) - i64::from(b))
 }
 
+fn encode_rmi(out: &mut Vec<u8>, indices: &[usize]) {
+    fn encode_byte(b: u8) -> u8 {
+        match b {
+            0..=25 => b + b'A',
+            26..=51 => b + b'a' - 26,
+            52..=61 => b + b'0' - 52,
+            62 => b'+',
+            63 => b'/',
+            _ => panic!("invalid byte"),
+        }
+    }
+
+    let mut data = [0u8; 8];
```

Comment:
A fixed size array is not suitable here. It would break when there are more than 64 mappings in a line